<launch>
    <!--- Sim Time -->
    <param name="/use_sim_time" value="false"/>

    <!--- Run Rviz  args="-d $(find test_ros)/launch/test.rviz" -->
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find sweep_robot)/launch/my.rviz"/>

    <node pkg="sweep_robot" type="stereo_vo_node" name="stereo_video_node" output="screen">
        <param name="CAM_ID" value="0" type="str"/>
        <param name="DOWN_SAMPLE" value="1" type="str"/>
        <param name="SAVE" value="1" type="str"/>
        <param name="GREY" value="0" type="str"/>
    </node>


    <node pkg="rosbag" type="record" name="bag_record" args="/imu0 /cam0/image_raw /cam1/image_raw"/>

</launch>